app/models/post.rb

```diff
@@ -0,0 +1,17 @@
+class Post < ApplicationRecord
+  belongs_to :user, class_name: 'User', foreign_key: 'author_id'
+  has_many :likes
+  has_many :comments
+
+  # Callback
+  after_save :updates_user_posts_counter
+
+  # Methods
+  def updates_user_posts_counter
+    user.update(post_counter: user.posts.count)
+  end
```

Comment:
[OPTIONAL]
- [ ] Same here as pointed in the previous comment :point_up_2:

Reply:
- Method made private.